src/dmd/parse.d

```diff
@@ -3497,6 +3498,28 @@ final class Parser(AST) : Lexer
             t = parseVector();
             break;
 
+        case TOK.traits:
```

Comment:
Also needs entry in `isBasicType()`, `parseMixin()`, `parseStatement()`, `parsePrimaryExp()` and `parseUnaryExp()`. Basically, just look at where `TOK.typeof_` appears, which is very similar.